source/install/linux.rst

```diff
@@ -281,20 +278,44 @@ Retrieve the latest available stable version from our `Github release page <http
 
 .. code-block:: bash
 
-   cd /var/www/
+   
    # Options
    VERSION=3.4.0
+   # chose location where download your zip (e.g. /var/www or your home)
+   LOCATION=/var/www 
    # Archive recovery with wget
+   cd $LOCATION
    wget https://github.com/3liz/lizmap-web-client/archive/$VERSION.zip
    # Unzip archive
    unzip $VERSION.zip
+
+
+Or clone the repository and than use a specific version (it is a better solution for future update)
+
+.. code-block:: bash
+
+   
+   # Options
+   VERSION=3.4.0
+   # chose location where create your clone (e.g. /var/www or your home)
+   LOCATION=/var/www 
+   
+   # Clone repository
+   cd $LOCATION
+   git clone https://github.com/3liz/lizmap-web-client.git
```

Comment:
This is not possible with Lizmap 3.4 : https://github.com/3liz/lizmap-web-client/blob/master/INSTALL.md#get-the-source  (I don't think people should build it, they should download packages)